Dockerfile

```diff
@@ -67,6 +67,7 @@ USER nextjs
 EXPOSE 3000
 
 ENV PORT=3000
+ENV NEXT_PUBLIC_BRANDING=sizhu
```

Comment:
Do not change content between "start of boilerplate" and "end of boilerplate" blocks